ReadExcelFile.py

```diff
@@ -0,0 +1,25 @@
+#This program will read in an Excel file and print the contents of a specific cell.
+
+#import openpyxl module function load_workbook
+from openpyxl import load_workbook
+
+#Define variable name for the workbook
+wb = load_workbook('samplebook.xlsx')
+
+#Define variable name for worksheet
+sheet = wb['Sheet1']
+
+#Define variable for the cell to read
+cell = sheet.cell(1,1)
+
+#Define varible for cell value to print
+my_val = cell.value
+
+#Print the cell value
+print(my_val)
+
+#Read in values of cells C1 through C5 and print
+for i in range(1,3):
+    C_Cell = sheet.cell(i,3)
+    C_Cell_value = C_Cell.value
+    print(C_Cell_value)
```

Comment:
Let's do a refactoring: since C_Cell_value is only used once, try to "inline" its use.

Inlining a variable means replacing all uses with the definition of the variable. For example, instead of saying
one = 1
two = one + 1

You could just say
two = 1 + 1

This replaces the use of the variable "one" with its definition which is "in line" with the rest of the usage (as opposed to in a completely separate place).